localstack/testing/pytest/in_memory_localstack.py

```diff
@@ -42,8 +42,9 @@ def pytest_addoption(parser: Parser, pluginmanager: PytestPluginManager):
 
 
 @pytest.hookimpl(tryfirst=True)
-def pytest_sessionstart(session: Session):
+def pytest_runtestloop(session: Session):
     if not session.config.option.start_localstack:
+        LOG.info("Configuration option to start LocalStack not set.")
```

Comment:
nit: should we maybe invert the info, i.e., only log IF it is set? otherwise you'll see this every time you run unit,cli, or bootstrap tests, and that may be a bit misleading.

Reply:
The reason this log was introduced, was that I had a hard time to debug why LS was not starting. In our case, because the config option was not set, but it could also have been that the plugin was not picked up, or some other reason.
If LocalStack would try to start, we should see that in the log anyway

Reply:
Discussed in slack, and agreed upon a message on module load, to log the plugin load.